mani_skill/envs/tasks/rotate_cube.py

```diff
@@ -338,13 +338,15 @@ def compute_dense_reward(self, obs: Any, action: Array, info: Dict):
             + object_rot_weight * object_rot_reward
         )
         total_reward = finger_reach_object_reward + pose_reward
-        total_reward[info["success"]] = 3
+        total_reward = total_reward.clamp(-15, 15)
```

Comment:
It's possible for the reward to go negative and also beyond 15/-15? This suggests to me the reward in Isaac may have been designed poorly and we can improve it.

Not urgent for now anyway, lgtm